internal/receiver/prometheusremotewritereceiver/client.go

```diff
@@ -65,8 +66,17 @@ func (prwc *MockPrwClient) SendWriteRequest(wr *prompb.WriteRequest) error {
 
 	ctx, cancel := context.WithTimeout(context.Background(), prwc.Timeout)
 	defer cancel()
-
-	err = prwc.Client.Store(ctx, compressed)
+	retry := 3
```

Comment:
Can this be moved to a `_test.go` file since it's not intended to be used by the actual component?